lnd.go

```diff
@@ -56,6 +56,21 @@ import (
 	"github.com/lightningnetwork/lnd/watchtower/wtdb"
 )
 
+const (
+	// Permissions used to create admin macaroon
```

Comment:
nit: godoc comment need to start with the name of the constant/variable. So this needs to be something like `// adminMacaroonPermissions is the file permission that is used for creating the admin macaroon.`

Also, should we call this `adminMacaroonFilePerms`? Or even `sensitiveFilePerms` because we could use this in other places?

Reply:
Good idea adding `File` there. I think the perfect version would be making it configurable per each macaroon, together with group. In such case it's better to keep the names separate. But frankly I don't need it and would much prefer us spending time on other PRs. :) Of course if someone has enough time and will, I'm not opposed.